.riot/requirements/11031a2.txt

```diff
@@ -2,45 +2,45 @@
 # This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    pip-compile --no-annotate --resolver=backtracking .riot/requirements/11031a2.in
+#    pip-compile --allow-unsafe --no-annotate .riot/requirements/11031a2.in
 #
 annotated-types==0.7.0
-anthropic==0.28.0
+anthropic==0.34.1
```

Comment:
nit: Just an idea that popped up... But it would be handy if the PR title also contained the specific version that's being used for latest, so it's easy to look back/find them.